source/LoadPanel.cpp

```diff
@@ -71,17 +71,19 @@ namespace {
 	string TimestampString(time_t timestamp)
 	{
 		pair<const char*, const char*> format = TimestampFormatString(Preferences::GetDateFormat());
-		stringstream ss;
+		// size_t: Size of format string (required by strftime)
```

Comment:
If the two lines below this aren't clear enough, we should come up with better names, rather than have this comment.
```suggestion
```